.github/workflows/server_compatibility.yaml

```diff
@@ -390,8 +390,10 @@ jobs:
       - name: Set HAZELCAST_SERVER_VERSION # required for client 4.1.0 to handle the bug.
         if: ${{ matrix.client_tag == 'v4.1.0' }}
         shell: bash
-        run: export HAZELCAST_SERVER_VERSION=${{ github.event.inputs.hz_version }}
+        run: echo "HAZELCAST_SERVER_VERSION=$HAZELCAST_SERVER_VERSION"
         working-directory: tag          
+        env:
+          HAZELCAST_SERVER_VERSION: ${{ github.event.inputs.hz_version }}
```

Comment:
Will this be visible in other steps? According to https://docs.github.com/en/actions/learn-github-actions/environment-variables#about-environment-variables, I feel like it will only be visible in this step

Reply:
Yeap, I saw the docs, and not %100 sure me either. Let's try this, if it's not working I'll change the level.

Reply:
Why don't we directly move this to the steps below directly? (`Run non-enterprise tests` and `Run all tests`)

Reply:
Removed the condition and set the version for each case.